### PR TITLE
ja: Update security translation

### PR DESCRIPTION
**Pull request by e-jigsaw**
ref: #2179

### CODE REVIEW

**Comment by e-jigsaw** [comment]
@nodejs/nodejs-ja Review plz!

(全体的に古くなっていたので書き換えました)

File: locale/ja/security.md [hunk 1]
```diff
@@ -2,98 +2,106 @@
...
 -->
-- このプロセスは他のプロジェクトの管理者との調整を必要とする際など、ときに時間を要することがあります。可能な限り迅速にバグを修正するために尽力するだけでなく、一貫性のあるプロセスで情報開示を行うために上記のリリースプロセスを従うこともまた重要です。
 
+## 情報開示のポリシー
+
+Node.js における、情報開示のポリシーをここに記します。
+
+- 脆弱性が報告されると、主要なメンバーがアサインされます。このメンバーは、修正とリリースのプロセスを調整します。脆弱性が確認されたら、影響のある全てのバージョンのリストを作成します。コードは、潜在的な同様の問題があるか精査されます。修正は、メンテナンスされている全てのバージョンに向けて用意されます。これらの修正は、正式なアナウンスを行うまでは公開リポジトリには追加されません。
+
+- CVE (Common Vulnerabilities and  Exposures (CVE®)) により、脆弱性に関する情報解禁日が提案され、それを決定します。
```
Comment (okuryu) [inline locale/ja/security.md]:
```suggestion
- CVE (Common Vulnerabilities and Exposures (CVE®)) により、脆弱性に関する情報解禁日が提案され、それを決定します。
```

File: locale/ja/security.md [hunk 2]
```diff
@@ -2,98 +2,106 @@
...
+
+- - 情報解禁日に、Node.js のメーリングリストに公式発表と同様のものが送信されます。修正済みのバージョンは、公開リポジトリに送信され、新しいビルドが nodejs.org に置かれます。メーリングリストに共有されてから6時間以内に、セキュリティ勧告を Node.js のブログにて行います。
```
Comment (okuryu) [inline locale/ja/security.md]:
```suggestion
- 情報解禁日に、Node.js のメーリングリストに公式発表と同様のものが送信されます。修正済みのバージョンは、公開リポジトリに送信され、新しいビルドが nodejs.org に置かれます。メーリングリストに共有されてから6時間以内に、セキュリティ勧告を Node.js のブログにて行います。
```

**Comment by ghost** [comment]
@nodejs/nodejs-ja：Any other clues or review suggestions?

**Review by okuryu — APPROVED** [review]
👍 looks good to me.

@Maledong thanks for your helps.

**Comment by e-jigsaw** [comment]
Thanks!

**Pull request merged**